examples/chart/teleport-kube-agent/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+name: teleport-kube-agent
+apiVersion: v2
+version: 0.0.1
+appVersion: 5.0.0-rc.2
```

Comment:
This should probably be 5.0.0, as it will be out by the time people are wanting to use this chart.